### PR TITLE
Updated cluster-install-docs

### PR DESCRIPTION
**Pull request by wombat**

### CODE REVIEW

File: cluster-install/readme.adoc [hunk 1]
```diff
@@ -55,10 +55,10 @@ https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-iam-user. https
 
 Kops needs a "`state store`" to store configuration information of the cluster.  For example, how many nodes in the cluster, the instance type of each node, and the Kubernetes version. The state is stored during the initial cluster creation. Any subsequent changes to the cluster are also persisted to this store. As of now, Amazon S3 is the only supported storage mechanism. Create an S3 bucket and pass that to the kops CLI during cluster creation.
 
-NOTE: The bucket name must be unique otherwise you will encounter an error on deployment. For demonstration purposes only, we will use an example bucket name of `example-state-store`.
+NOTE: The bucket name must be unique otherwise you will encounter an error on deployment. We will use an example bucket name of `example-state-store-` and add a randomly generated string to the end.
 
     # create variables for bucket, state store and cluster names
-    export S3_BUCKET=example-state-store
+    export S3_BUCKET=example-state-store-$(cat /dev/random | LC_CTYPE=C tr -dc "[:alpha:]" | tr '[:upper:]' '[:lower:]' | head -c 32)
```
Comment (arun-gupta) [inline cluster-install/readme.adoc]: Will this command work on Windows?

Comment (wombat) [reply cluster-install/readme.adoc]: I had no way to test this but I guess not except they use a vm to work in

Comment (arun-gupta) [reply cluster-install/readme.adoc]: can we create a platform-agnostic command?

Comment (wombat) [reply cluster-install/readme.adoc]: Do the export commands work at all under windows?

Comment (arun-gupta) [reply cluster-install/readme.adoc]: Good point, `kops` is only Mac and Linux anyway. So this would be fine.

**Pull request merged**